src/main/java/bg/codexio/springframework/data/jpa/requery/resolver/FilterJsonArgumentResolver.java

```diff
@@ -81,28 +83,41 @@ public Object resolveArgument(
             ModelAndViewContainer mavContainer,
             @NotNull NativeWebRequest webRequest,
             WebDataBinderFactory binderFactory
-    ) throws Exception {
-        var filterJson = webRequest.getParameter("filter");
-        var complexFilterJson = webRequest.getParameter("complexFilter");
-
+    ) {
+        var request = webRequest.getNativeRequest(HttpServletRequest.class);
         var genericType =
                 (Class<?>) ((ParameterizedType) parameter.getGenericParameterType()).getActualTypeArguments()[0];
-
-        if (filterJson != null) {
-            return this.getSimpleFilterSpecification(
-                    filterJson,
-                    genericType
-            );
-        }
-
-        if (complexFilterJson != null) {
-            return this.getComplexFilterSpecification(
-                    complexFilterJson,
-                    genericType
-            );
-        }
-
-        return this.noFilterSpecification();
+        this.logger.debug(
+                "start traversing active adapters lost with size {}. If no "
```

Comment:
Not sure what `active adapters lost` means. Also it's not correct that not logs will appear if no adapters support, because at least `Invoking ...` line will appear. Reduce this message to `{} active adapters will be tested against the request`

Reply:
done